tests/test_timer.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
```

Comment:
Curious - weren't there any Timer tests before this? 😱

Reply:
there were not any timer tests before.